Neos.Flow/Classes/Log/ThrowableStorage/FileStorage.php

```diff
@@ -266,8 +266,8 @@ protected function cleanupThrowableDumps(): void
         if ($this->maximumThrowableDumpAge > 0) {
             $cutoffTime = time() - $this->maximumThrowableDumpAge;
 
-            /** @var \SplFileInfo $directoryEntry */
             $iterator = new \DirectoryIterator($this->storagePath);
+            /** @var \DirectoryIterator $directoryEntry */
```

Comment:
```suggestion
```

Not necessary imho. phpstan should figure this out on its own. (But does it even care on level 2?)